stdpopsim/catalog/EscCol/species.py

```diff
@@ -32,8 +32,7 @@
             # Wielgoss et al. (2011) calculated for strain REL606,
             # from synonymous substitutions over 40,000 generations.
             mutation_rate=8.9e-11,
-            recombination_rate=0.0,
-            gene_conversion_rate=8.9e-11,
+            recombination_rate=8.9e-11,
             gene_conversion_length=345,
         )
```

Comment:
There is also no citation associated with the average tract length of gene conversion. @fbaumdicker referred me again to [Didelot et al. (2012)](https://pubmed.ncbi.nlm.nih.gov/22712577/), which quotes an average of 542. This should probably be re-checked.